client/blocks/jetpack-backup-creds-banner/index.jsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Banner from 'components/banner';
+import getRewindState from 'state/selectors/get-rewind-state';
+import QueryRewindState from 'components/data/query-rewind-state';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug, isJetpackSite } from 'state/sites/selectors';
+
+class JetpackBackupCredsBanner extends Component {
+	static propTypes = {
+		// Banner location, passed to tracks event, single word to follow tracks naming conventions.
+		location: PropTypes.string.isRequired,
+	};
+
+	render() {
+		const { isJetpack, rewindState, siteId, slug, translate } = this.props;
+		const location = this.props.location || 'nolocation';
+		const event = `calpyso_jetpack_backup_credential_banner_${ location }_click`;
+		return (
+			<Fragment>
+				{ siteId && isJetpack && <QueryRewindState siteId={ siteId } /> }
```

Comment:
Seems like this queries the Jetpack rewind state only for Jetpack sites. So the next block of code won't be executed for non-Jetpack sites.

To make this more obvious, should we have:

```
if ( isJetpack ) {
	return null;
}
```

above this `return <Fragment>...` statement?

Reply:
Yes, agree, it is not explicit enough currently. I'll change this.